packages/core/test/tools/rpc-client.js

```diff
@@ -16,17 +16,17 @@ describe('rpc client', () => {
 
   it('should handle an explosion', () => {
     // mock 3 explosions due to retry
-    mocky.mockRpcFail('this is an expected explosion');
-    mocky.mockRpcFail('this is an expected explosion');
-    mocky.mockRpcFail('this is an expected explosion');
+    mocky.mockRpcFail('this is an expected explosion', 500);
```

Comment:
The `explode` RPC method actually throws an HTTP 500 via an exception being raised 😅